libr/core/cmd_debug.c

```diff
@@ -1651,6 +1651,20 @@ static int cmd_debug_map(RCore *core, const char *input) {
 					if (libname) {
 						char *cmd, *res;
 						const char *file = map->file? map->file: map->name;
+						char *newfile = NULL;
+						if (!r_file_exists (file)) {
+							newfile = r_file_temp ("memlib");
+							if (newfile) {
+#if __WINDOWS__ && !__CYGWIN__
+								const char *nul = "nul";
+#else
+								const char *nul = "/dev/null";
+#endif
+								file = newfile;
+								r_core_cmdf (core, "wtf %s 0x%"PFMT64x" @ 0x%"PFMT64x" 2> %s",
+								             file, map->size, baddr, nul);
+							}
+						}
 						if (symname) {
 							cmd = r_str_newf ("rabin2 %s-B 0x%08"PFMT64x" -s %s | grep %s", mode, baddr, file, symname);
```

Comment:
command injection vulns in here

Reply:
huh how? the `file` for the `wtf` is completely under the code's control

Reply:
though it's probably best to call the appropriate functions instead

Reply:
ideally we should use the C api instead of calling commands.

the symname is under control too?

Reply:
the rabin2 code is not mine :p ... for that line, afaik the symname comes from the user but the `file` isn't under control